CHANGELOG.md

```diff
@@ -1,3 +1,23 @@
+# 1.0.9.1 (2020-05-04)
```

Comment:
Are we releasing this today?

Reply:
I'm starting the process today. 
Since it's May 4th I'd like to take the opportunity. 🛰️

Reply:
Haha ok. I think we should change it to be accurate though when we know the release date